public/modules/articles/articles.viz.js

```diff
@@ -499,6 +515,9 @@ var valign, stacked, articlesScope, aligned = true;
                                         .popover(function () {
                                             return section_hover(d, curS)
                                         })
+                                        .on("mouseenter", function() {
+                                            $(".popover").addClass("articles-popover");
+                                        })
                                         .text(clean_premier(titre_section(sub_section(curS), longlabel)));
                                     ct++;
```

Comment:
I'm a bit annoyed by all this mouseenter events added to the existing popover just to add a class saying the popover is the one of the articles view. Actions on articles view were already quite slow and my browser seems in pain whenever moving the mouse on articles now. Couldn't all of these be handled better (less event handling) simply in css by having a .popover under the class of the current view (probably meaning moving the popover div from index to each module template)?